frontend/src/Hooks/useGameState.js

```diff
@@ -211,12 +211,21 @@ export default function useGameState() {
         }
 
         function onDayStart({ timeToVote }) {
-            dispatch({
-                type: 'day-start',
-                status: 'Select someone to be on trial',
-                votablePlayers: state.alivePlayers.filter((p) => p !== generalState.nickname),
-                timeToVote,
-            });
+            const amIDead = !state.alivePlayers.includes(generalState.nickname);
```

Comment:
Can we make amIdead another state value too? That way we can check for if you're dead easily in other components too e.g. I would like to hide my abstain vote button if dead.

Also would this only prevent players from voting during the day? Seems like it could allow people to still vote during the night and during a trial.... But this is just from me reading code

Reply:
@reeved Right.. Looks like it. I am not that familiar with the mafia game.. haha .. We do have another version but it does not have a trial stage.. lol.. going to fix that real quick